### DISCUSSION TITLE
Type Error

### DISCUSSION THREAD

**Discussion by mrvicthor** [prisma/prisma#22374] [Q&A]
### Question

Hi @prismaemployees, I encountered this issue `./node_modules/@prisma/client/runtime/library.d.ts:1325:35
Type error: '?' expected`. during deployment on vercel after pushing my changes to github. I was following this NextJs tutorial in this link https://vercel.com/guides/nextjs-prisma-postgres, and I encountered this issue after completeing step 5 and pushing my changes to github. I have tried updating prisma by running this command in the following order.

```
npm install @prisma/cli@latest   ,
npm uninstall @prisma/cli 
npm install prisma --save-dev   
npx prisma
npm install @prisma/client  
```
However, I still encounter the same issue. Below is an image of the error message on vercel.

<img width="1194" alt="Screenshot 2023-12-13 at 00 37 34" src="https://github.com/prisma/prisma/assets/43174827/026c41e1-a623-474e-a0be-52ecaf3033de">

Please how can i resolve this

### Information about Prisma Schema, Client Queries and Environment (optional)

```prisma
// Add your schema.prisma
```

```ts
// Add any relevant Prisma Client queries here
```

- OS: 
- Database: 
- Node.js version: 

- Run `prisma -v` to see your Prisma version and paste it

**Comment by ludralph** [comment]
Hi @mrvicthor 👋 

Thank you for raising this question.

Another user recently ran into a similar error in this Github [discussion](https://github.com/prisma/prisma/discussions/21861). Can you please update your version of Typescript to version 5.2.2 as suggested in that discussion? You may also need to update your version of Node.js.

If this answers your question, it would be great if you could **mark this Discussion as answered** to indicate that it has been resolved. 

Otherwise please let us know how else we can help you further or close the Discussion if it was resolved in some other way 🙏

**Reply by mrvicthor** [reply]
Hi @ludralph,
Thanks for the suggestion. i will have a look.

**Reply by garrettreid** [reply]
This solved the same issue for me.

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that haven’t had any recent activity and appear to be outdated.

If this discussion is still important to you or unresolved, we’d love to hear from you! Feel free to reopen it or start a new one with updated details.

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and being part of the community!
